### PR TITLE
ISSUE #41: This adds the certPath configuration and also allows docke…

### PR DESCRIPTION
**Pull request by ysb33r**
…rHost to assume a default value from system property or environmental variable

### CODE REVIEW

**Comment by ysb33r** [comment]
Not sure if those failure are related to my PR unless it the change to `dockerHost` default value that I made.

File: src/main/groovy/de/gesellix/gradle/docker/DockerPluginExtension.groovy [hunk 1]
```diff
@@ -1,9 +1,44 @@
...
+    }
+
+    /** Returns the certificate path as an abolute path to the certificate.
+     * If {@code certPath} was not configured it will attempt to check for system property
+     * {@code docker.cert.path} first and then for the {@code DOCKER_CERT_PATH} environment variable.
+     *
+     * @return Absolute path as a string or null.
+    */
+    String getCertPath() {
+      certPath ? project.file(certPath).absolutePath : null
```
Comment (gesellix) [inline src/main/groovy/de/gesellix/gradle/docker/DockerPluginExtension.groovy]: Where does the new property have an effect? I didn't find code which passes the `certPath` down to the docker-client library.

**Comment by gesellix** [comment]
I'll check the ci build. Looking at the code I also don't see a reason for it to fail.

File: src/main/groovy/de/gesellix/gradle/docker/DockerPluginExtension.groovy (hunk 1 above)
Comment (ysb33r) [inline src/main/groovy/de/gesellix/gradle/docker/DockerPluginExtension.groovy]: I have not hooked that in as yet. I just wanted to get the up front DSL parts in there for you to see. Eiterh you can add that bit or I can have a quick look.

Comment (gesellix) [inline src/main/groovy/de/gesellix/gradle/docker/DockerPluginExtension.groovy]: alright, I'll do it then. There's probably some work on the docker-client necessary for it to work, and I guess you didn't want to go down that rabbit hole ;-)

**Comment by gesellix** [comment]
I merged it manually with some minor layout changes, the tests ran successfully on my local setup.
Thanks!

**Pull request merged**